src/function.ts

```diff
@@ -783,6 +799,24 @@ export class CallbackLambdaCode extends aws_lambda.Code {
               return o;
             };
 
+            /**
+             * When the StreamArn attribute is used in a Cfn template, but streamSpecification is
+             * undefined, then the deployment fails. Lets make sure that doesn't happen.
+             */
+            const transformBus = (o: CfnResource): CfnResource => {
```

Comment:
Is this a bus? Docs hint that it is a DynamoDB Strean?

Reply:
err, yes, also don't need that. Removed